config/default.json

```diff
@@ -6,9 +6,6 @@
     "wsUrl": "ws://127.0.0.1:8546",
     "wsTimeToLive": 120000
   },
-  "coincap": {
-    "retries": 5
```

Comment:
This config setting is used in `src/eth-to-usd.js:7` so should not be removed from any config file. Name should be changed as it no longer is related to CoinCap.

Reply:
@gabmontes sorry I cleaned up coincap key-value after I my testing and didnt realized retry is used there.  Commit new code. This should work fine.